Mithril update 2023-10-25 (#278)
input-output-hk/cardano-updates: blog/2023-10-25-mithril.md

---
title: Mithril Team Update
slug: 2023-10-25-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> The Mithril team has released a new distribution [`2342.0`](https://github.com/input-output-hk/mithril/releases/tag/2342.0), which includes the following enhancements: support for SPO tickers displayed in the Mithril explorer, and support for producing slim Docker images for the Mithril nodes.

Reviewer: ```suggestion
This week, the Mithril team released a new distribution [`2342.0`](https://github.com/input-output-hk/mithril/releases/tag/2342.0). This release includes support for SPO tickers displayed in the Mithril explorer and the ability to produce slim Docker images for Mithril nodes.
```

> They also kept working on the decentralization of the Mithril networks with a proof of concept of peer-to-peer (P2P) networking between nodes. The team kept working on the adaptation of the Mithril client as a library, and on the deterministic computation of the UTxO set from the immutable files. Additionally, they fixed the bottleneck of the key registration in the aggregator.

Reviewer: ```suggestion
They also kept working on decentralizing the Mithril networks with the peer-to-peer (P2P) networking proof of concept, progressed with the adaptation of the Mithril client as a library, and worked on the deterministic computation of the UTXO set from the immutable files. 
```

> Finally, they investigated some unreachability on the `release-mainnet` aggregator, and they enhanced the UI of the Mithril explorer.

Reviewer: ```suggestion
Finally, they fixed the bottleneck of the key registration in the aggregator, investigated some unreachability on the `release-mainnet` aggregator, and enhanced the Mithril explorer UI.
```
